scripts/install_from_scratch.sh

```diff
@@ -51,6 +72,10 @@ EOF
 # Create tables
 python -m scripts.initialize_db
 
+# Add simple lookup tables, since it's easy for a user to miss these later.
+python -m ambuda.seed.lookup.page_status
```

Comment:
Aren't these already loaded with `make db-seed-basic`? Should we just run `make db-seed-basic`, perhaps, in `make install`?

Reply:
Yes, they are. The difference is that `make db-seed-basic` generally fetches large resources over the network, whereas the `lookup` tables are defined in the application itself and are basically glorified enums.

I suppose the deeper question is what we want `make install` to be. If it's just to quickly build a working setup, running `db-seed-basic` makes sense. If we expect this to be one stage in a larger build process, then I would prefer to focus mainly on application logic and the small lookup tables that are are essentially application logic.

I have a slight preference to the second, hence the setup here. But I'm open to being convinced otherwise. What do you suggest?

Reply:
I don't think that the application logic-data logic distinction is more important than convenience / ease of setup. I do see your point about fetching large resources over the network though. In the long term (perhaps we can make this an issue if we decide to do it), I think I would prefer creating something like `make db-seed-tiny` that doesn't fetch large resources over the network, but just loads a single small text, perhaps from a fixture stored in the same github repository.

Reply:
Makes sense to me! I'll just run `make db-seed-basic` here.